### PR TITLE
I think this might change the dt=0 problem?

### PR DESCRIPTION
**Pull request by ntmoore**
there was a vx*dt problem in the position update.  Also  t_now = millis() was missing.  compiles, but haven't tried this with hardware.

### CODE REVIEW

File: Lab 5 #1 [hunk 1]
```diff
@@ -53,69 +53,62 @@ void setup() {
...
-    
-    Serial.print(delta_t);
-  // Serial.print('\t');
-   // Serial.print(ay);
-   //Serial.print('\t');
-   // Serial.println(az);
-  }
-  
- delta_t = t_now - t_old;
+    t_now = millis();
```
Comment (ntmoore) [inline Lab 5 #1:59]: I think this was the line that was really causing problems.  All of the other changes are mostly formatting and data types.

File: Lab 5 #1 [hunk 2]
```diff
@@ -53,69 +53,62 @@ void setup() {
...
+    delta_t = (1.0/1000.0)*(float)(t_now - t_old);
```
Comment (ntmoore) [inline Lab 5 #1:60]: Note, convert from ms to s

File: Lab 5 #1 [hunk 3]
```diff
@@ -53,69 +53,62 @@ void setup() {
...
-  lcd.print("Dz: ");
-  lcd.print(z);
-  lcd.print("");
-
-  ax=ax_old;
-  ay=ay_old;
-  az=az_old;
+
+    // update position
+    delta_x = (0.5 * ax * (delta_t*delta_t)) + (vx * delta_t);
```
Comment (ntmoore) [inline Lab 5 #1:74]: without the *delta_t added in the units don't work.

File: Lab 5 #1 [hunk 4]
```diff
@@ -21,21 +21,21 @@
 const int rs = 13, en = 12, d4 = 11, d5 = 10, d6 = 9, d7 = 8;
 LiquidCrystal lcd(rs, en, d4, d5, d6, d7);
 int position_pin = 6;
-float t_now = millis();
+unsigned long t_now = millis();
```
Comment (ntmoore) [inline Lab 5 #1:24]: this change of type is probably helpful.

**Review by go6721of — APPROVED** [review]

**Pull request merged**